### PR TITLE
Package lutils.1.51.2

### PR DESCRIPTION
**Pull request by jahierwan**
### `lutils.1.51.2`
Tools and libs shared by Verimag/synchronous tools (lustre, lutin, rdbg)
The lutils library contains various modules shared between tools developped at
Verimag in the synchronous group. Those modules deal with:
- generate and parse RIF files
- generate dro files (to call luciole)

The gnuplot-rif tool vizualise RIF files using gnuplot.

---
* Homepage: https://gricad-gitlab.univ-grenoble-alpes.fr/verimag/synchrone/lutils/
* Source repo: git+https://gricad-gitlab.univ-grenoble-alpes.fr/verimag/synchrone/lutils.git
* Bug tracker: https://gricad-gitlab.univ-grenoble-alpes.fr/verimag/synchrone/lutils/issues

---
:camel: Pull-request generated by opam-publish v2.0.3

### CODE REVIEW

**Comment by avsm** [comment]
We generally don't delete older versions of packages published within opam. Would it be possible to restore that package -- we can pull the archive out of the mirrors.

**Comment by jahierwan** [comment]
> We generally don't delete older versions of packages published within opam. Would it be possible to restore that package -- we can pull the archive out of the mirrors.

Actually it is! (the checksum in the same). I've put it back in my pool/ dir

**Comment by mseri** [comment]
Looks good now, thanks a lot!

**Comment by jahierwan** [comment]
Thank you !

File: packages/lutils/lutils.1.51.2/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "ocaml" {>= "4.02"}
+  "dune"  {>= "2.0"}
+  "base-unix"
+  "ocamlfind"
+  "num"
+]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ]
+install: [make "install"]
```
Comment (kit-ty-kate) [inline packages/lutils/lutils.1.51.2/opam:21]:
```suggestion
```
Your makefile script calls dune install underneath. This is unnecessary with opam as the build step already creates an .install file that is going to be used by opam.

**Comment by jahierwan** [comment]
ok, thanks for the tip. I'll remove it for future releases.

**Pull request merged**